nestor-cervantes.txt

```diff
@@ -9,26 +9,25 @@
     blueprint for any website.
 
     3. What is an attribute and where do we put it?
-    An attribute gives additionally information to a tag, such as an image source or a
+    An attribute gives additional information to a tag, such as an image source or a
     clas name or ID.
 
     4. What is the h1 tag used for? How many times should I use it on a page?
     The h1 is used to mark the main header of a website.  It should only be used once.
     
-    5. Name two tags that have required attributes
+    5. Name two tags that have required attributes?
     both img and a tags require attributes (src and href respectively).
 
     6. What do we put in the head of our HTML document? 
-    This is where all the meta data for an HTML document goes.  Tags that link to your
-    external css and javascript files, for example.
+    This is where all the metadata for an HTML document goes.  Tags that link to your
+    external CSS and javascript files, for example.
 
     7. What is an id? 
-    An id is a name that you give an element within it's opening tag that will help
-    identify it.  Unlike a class, each element can only have one unique ID.  The same
+    An id is a name that you give an element within it's opening tag that will help identify it.  Unlike a class, each element can only have one unique ID.  The same
     id name cannot be used for more than one element.
 
     8. What elements can I add an id to? 
-    an ID can be added to any html element.
+    an ID can be added to any HTML element.
```

Comment:
love how you took the time to go back through and change your wording or spelling. That attention to detail will go a long way